api/src/shared/common/db_utils.py

```diff
@@ -168,12 +167,24 @@ def get_gtfs_rt_feeds_query(
         joinedload(Gtfsrealtimefeed.gtfs_feeds),
         *get_joinedload_options(),
     )
-    if is_official:
-        feed_query = feed_query.filter(Feed.official)
+    feed_query = add_official_filter(feed_query, is_official)
+
     feed_query = feed_query.limit(limit).offset(offset)
     return feed_query
 
 
+def add_official_filter(query: Query, is_official: bool) -> Query:
```

Comment:
[picky]: allow None to be passed as a parameter
```suggestion
def add_official_filter(query: Query, is_official: bool | None) -> Query:
```